nvme.c

```diff
@@ -358,11 +349,10 @@ static int get_telemetry_log(int argc, char **argv, struct command *cmd, struct
 		}
 		offset += bs;
 	}
-	free(page_log);
- free_hdr:
+	goto free_mem;
```

Comment:
What's with the goto the very next line?! That just looks weird.

Reply:
If we don't free page_log and hdr, there will be a memory leak. Beforehand only page_log was being freed at the end. Because there is already a goto that does this (free_mem), I thought it would be better to use it rather than typing out `free(page_log); free(hdr);`.

Reply:
The code execution just calls through the label, though. You don't need to 'goto' the very next line of code as that's going to happen anyway.

Reply:
Ah got it. I thought the label was only called when you wrote goto. Sorry about that. Removed.